front/src/modules/ui/navigation/menu-item/components/MenuItem.tsx

```diff
@@ -19,7 +19,7 @@ export type MenuItemProps = {
   LeftIcon?: IconComponent | null;
   accent?: MenuItemAccent;
   text: string;
-  iconButtons?: MenuItemIconButton[];
+  iconButtons?: MenuItemIconButton[] | null;
```

Comment:
is it working with the current API if we pass an empty array? I feel a null option is duplicated with []